### DISCUSSION TITLE
Where is scoop executable?

### DISCUSSION THREAD

**Discussion by Geobert** [ScoopInstaller/Scoop#6027] [Help and Questions]
Hi, 

I’m facing an issue when NordVPN is up, `install` command gives:
```
The requested address is not valid in its context. 
```

And if I pause the VPN, it works. So I wanted to put scoop in the “no vpn” list of splitted networking but I can’t find the `.exe`

**Comment by niheaven** [comment]
It's a PowerShell script that located at `scoopdir\apps\scoop\current\bin\scoop.ps1`

**Reply by Geobert** [reply]
Thanks for your answer. It seems NordVPN is expecting only a binary :(
